chart/templates/mongodb-statefulset.yaml

```diff
@@ -54,11 +54,19 @@ spec:
         - name: mongodb
           containerPort: 27017
         volumeMounts:
-        - name: data
+        - name: mongodb-pv-storage
           mountPath: /bitnami/mongodb
           subPath: 
         resources:
           {}
-      volumes:
-      - name: data
-        emptyDir: {}
\ No newline at end of file
+  volumeClaimTemplates:
+  - metadata:
+      name: mongodb-pv-storage
+      labels:
+        {{- include "movies.labels" . | nindent 4 }}
```

Comment:
@RinkiyaKeDad you need to do `nindent 8` to render it properly

Reply:
Thanks! That fixed it. The volume is coming up now:
![Screen Shot 2021-12-01 at 12 39 56 PM](https://user-images.githubusercontent.com/56963264/144188233-bfca8392-728a-4bed-a113-7dbfdab222ac.png)